Docs updates (#1201) (#1358)
open-telemetry/opentelemetry.io: content/en/docs/concepts/otel-concepts.md

provide statistical information in aggregate. Some examples of use cases for
metrics include:

- Reporting the total number of bytes read by a service, per protocol type.
- Reporting the total number of bytes read and the bytes per request.
- Reporting the duration of a system call.
- Reporting request sizes in order to determine a trend.
- Reporting CPU or memory usage of a process.
- Reporting average balance values from an account.
- Reporting current active requests being handled.

> For more information, see the [metrics specification][].

## Logs in OpenTelemetry

A **log** is a timestamped text record, either structured (recommended) or
unstructured, with metadata. While logs are an independent data source, they may
also be attached to spans. In OpenTelemetry, any data that is not part of a
distributed trace or a metric is a log. For example, _events_ are a specific
type of log. Logs are often used to determine the root cause of an issue and
typically contain information about who changed what as well as the result of
the change.

> For more information, see the [logs specification][].

## Baggage in OpenTelemetry

> Imagine you wanted to have the CustomerId appear on all your spans, but it’s
> only available on the initial API request because your Stock Check API doesn’t
> need a Customer context. This is where OpenTelemetry Baggage comes to the
> rescue.
> 
> In OpenTelemetry, "Baggage" refers to contextual information that’s passed
> between spans. More specifically, it’s about passing that context between
> service boundaries. So really, it’s about pushing that context over an HTTP,
> gRPC, or a message so the other service can use it to add context to its span.

Reviewer: This is a good scenario, but I found it a little confusing to read. I'd suggest a re-wording like so:

```suggestion
Imagine you want to have a `CustomerId` attribute on every span in your trace,
but it's only available in one specific service. To accomplish your goal, you can
use OpenTelemetry Baggage to propagate this value across your system.

In OpenTelemetry, "Baggage" refers to contextual information that’s passed
between spans. It's a key-value store that resides within a trace context,
making values available to any span created within that trace.
```

> OpenTelemetry uses [Context Propagation](#context-propagation) to pass this
> concept around, and each of the different library implementations has
> propagators that will parse and make that Baggage available without you needing
> to explicitly implement it.

Reviewer: ```suggestion
OpenTelemetry uses [Context Propagation](#context-propagation) to pass Baggage
around, and each of the different library implementations has
propagators that will parse and make that Baggage available without you needing
to explicitly implement it.
```

> ![OTel Baggage](/img/otel_bagage.png)

Reviewer: This imagine doesn't seem to have anything about baggage in it?

on development teams to build helpers in every framework and language, which can
unintentionally get neglected when other higher-priority items come up.

### What OTel Baggage be used for?

OTel Baggage should be used for non-sensitive data that you're okay with
potentially exposing to third parties.

Common use cases include information that’s only accessible further up a stack.
This can include things like Account Identification, User Ids, Product Ids, and
origin IPs, for example. Passing these down your stack allows you to then add
them to your Spans in downstream services to make it easier to filter when
you’re searching in your Observability back-end.

There are no built-in integrity checks to ensure it was your Baggage items, so
exercise caution when working with Baggage.

![OTel Baggage](/img/otel_baggage-2.png)

### Baggage != Span attributes

One important thing to note about Baggage is that it is not a subset of the
[Span Attributes](#attributes). When you add something as Baggage, it does not
automatically end up on the Attributes of the child system’s spans. You must
explicitly take something out of Baggage and append it as Attributes.

> ```javascript

Reviewer: ```suggestion
```csharp
```
